src/libaktualizr/uptane/uptane_network_test.cc

```diff
@@ -25,10 +25,8 @@
 Config conf("tests/config/basic.toml");
 std::string port;
 
-bool doTestInit(StorageType storage_type, const std::string &device_register_state,
-                const std::string &ecu_register_state) {
+bool doTestInit(const std::string &device_register_state, const std::string &ecu_register_state) {
   TemporaryDirectory temp_dir;
-  conf.storage.type = storage_type;
```

Comment:
You might consider leaving the storage_type option here for now since we know that we may have to support android storage sooner or later.